app/helpers/pagination_helper.rb

```diff
@@ -16,7 +16,7 @@ def per_page_options(options = [5, 10, 15, 25, 50])
   def react_pagination_props(collection = nil, classname = nil)
     {
       viewType: 'table',
-      itemCount: collection.total_entries,
+      itemCount: collection.klass.count,
```

Comment:
Wouldn't this trigger a `COUNT` query on the database every time, as well as ignore the actual collection with the permissions (e.g. in audits this would be Audit.count)?